src/server/api/routers/auth.ts

```
@@ -2,6 +2,7 @@ import { TRPCError } from '@trpc/server'
 import { getTranslations } from 'next-intl/server'
 // import { hash, compare } from 'bcryptjs' // TODO: Remove if not needed
 import { z } from 'zod'
+import crypto from 'crypto'
```

ESLint

[error] `crypto` import should occur before import of `@trpc/server`

(import/order)